### PR TITLE
Fixed ContentWithSize using correct header name

### LINKED ISSUE #293: ContentWithSize uses incorrect HTTP header
Helper class `ContentWithSize` which extracts the size from HTTP headers uses incorrect header `content-size` instead of `content-length`.

@artipie/contributors FYI

### PR DESCRIPTION
**Pull request by g4s8**
Fixes #293 - using `content-length` instead of `content-size` in
ContentWithSize class, added a test.

### CODE REVIEW

**Comment by g4s8** [comment]
@artipie/contributors could you please check PR? It's critical for Docker-adapter

File: src/test/java/com/artipie/http/slice/ContentWithSizeTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.artipie.http.Headers;
+import com.artipie.http.headers.ContentLength;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for {@link ContentWithSize}.
+ * @since 0.7
+ */
```
Comment (genryxy) [inline src/test/java/com/artipie/http/slice/ContentWithSizeTest.java:37]: @g4s8 looks good, except value of `@since` tag

Comment (g4s8) [reply src/test/java/com/artipie/http/slice/ContentWithSizeTest.java]: @genryxy thanks, fixed

**Review by genryxy — APPROVED** [review]
@g4s8 thanks, approve

**Pull request merged**